currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -89,3 +91,8 @@ def startElement(self, name, attrs):
                     (self.date_to is None or self.date <= self.date_to) and \
                     currency in self.currencies:
                 self.content[self.date.isoformat()][currency] = rate
+
+def parse_ecb_response(response, handler):
```

Comment:
Please don't make general methods outside the class, and anyway, being only 3 lines and not being reusable due to the scope, just put these 3 lines in the previous method.